core/src/main/scala/com/mesosphere/usi/core/CallerThreadExecutionContext.scala

```diff
@@ -0,0 +1,11 @@
+package com.mesosphere.usi.core
+
+import java.util.concurrent.Executor
+import scala.concurrent.ExecutionContext
+import scala.concurrent.ExecutionContextExecutor
+
+private[usi] object CallerThreadExecutionContext {
```

Comment:
We already have one `com.mesosphere.usi.async.CallerThreadExecutionContext`

Reply:
Does metrics-dropwizard already depend on core? Maybe we can move it from metrics-dropwizard to core and update the reference in metrics-dropwizard

Reply:
> I can't fathom a reasonable common module both the `core` and `metrics-dropwizard` have to share this same thread exec context.

yeah, we'll need something like `utils` package in the future. this duplication is ok for now

Reply:
Can we add a comment with a link to a JIRA or just a comment? I feel this will never be addressed and will confuse future maintainers.